mmv1/provider/terraform.go

```diff
@@ -328,7 +328,11 @@ func (t Terraform) getCommonCopyFiles(versionName string, generateCode, generate
 func (t Terraform) getCopyFilesInFolder(folderPath, targetDir string) map[string]string {
 	m := make(map[string]string, 0)
 	filepath.WalkDir(folderPath, func(path string, di fs.DirEntry, err error) error {
-		if !di.IsDir() && !strings.HasSuffix(di.Name(), ".tmpl") && !strings.HasSuffix(di.Name(), ".erb") {
+		if !di.IsDir() && !strings.HasSuffix(di.Name(), ".tmpl") && !strings.HasSuffix(di.Name(), ".erb") { // Exception files
+			if di.Name() == "gha-branch-renaming.png" || di.Name() == "clock-timings-of-branch-making-and-usage.png" {
```

Comment:
We had new files pushed here yesterday: https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/.teamcity/docs/images

The go engine copies these files over in the GA provider, where the Ruby engine does not.

We don't have logic that explicitly recreates https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/provider/terraform/common~copy.yaml#L179 so I'm opting to just start an exception list

Reply:
Good catch.

Reply:
we should consider a to-do to have a more organized file list here rather than a line of OR switches

Reply:
Yeah, I am going to do that after the switchover.